test.js

```diff
@@ -1,17 +1,36 @@
 import test from 'ava';
 import sortKeys from '.';
 
+const orderedDeepEqual = (t, a, b) => {
+	t.deepEqual(a, b);
+	t.deepEqual(Object.keys(a), Object.keys(b));
+};
+
 test('sort the keys of an object', t => {
 	t.deepEqual(sortKeys({c: 0, a: 0, b: 0}), {a: 0, b: 0, c: 0});
 });
 
 test('custom compare function', t => {
-	const compare = (a, b) => a.localeCompare(b);
-	t.deepEqual(sortKeys({c: 0, a: 0, b: 0}, {compare}), {c: 0, b: 0, a: 0});
+	const compare = (a, b) => -a.localeCompare(b);
+	orderedDeepEqual(t, sortKeys({c: 0, a: 0, b: 0}, {compare}), {c: 0, b: 0, a: 0});
+});
+
+test('custom compare function with context', t => {
```

Comment:
Also need a test for it in combination with the `deep` option.

Reply:
👍